package.json

```diff
@@ -36,6 +36,7 @@
     "history": "^3.0.0",
     "html-to-text": "^2.1.0",
     "immutability-helper": "^2.0.0",
+    "import": "0.0.6",
```

Comment:
@xavcz This import is not needed and I did not add it to the package.json. I merged with devel first and thought it came from there.